example_test.go

```diff
@@ -52,3 +53,25 @@ func Example() {
 	// http_request_seconds_created{status="200"} 1515151515.757576
 	// # EOF
 }
+
+func ExampleExemplar() {
+	reg := openmetrics.NewConsistentRegistry(mockNow) // or, openmetrics.DefaultRegistry()
+	internalError := reg.Counter(openmetrics.Desc{
+		Name: "internal_error",
+		Help: "A counter example",
+	})
+
+	_, err := os.Stat("/.dockerenv")
+	if err != nil {
+		// the combined length of label names and values of MUST NOT exceed 128 characters
+		errmsg := err.Error()
+		if len(errmsg) > 120 {
+			errmsg = errmsg[:120]
+		}
+
+		internalError.With().AddExemplar(&openmetrics.Exemplar{
```

Comment:
Would be nice to document `With(ARG)` if easy. Just so people have full example.

Reply:
it is documented in the example above